taxonomy/urls.py

```diff
@@ -1,19 +0,0 @@
-"""
-URLs for taxonomy app
-"""
-from __future__ import unicode_literals
-from django.conf.urls import url
-
-from taxonomy.views import (
-    create_vocabulary,
-    edit_vocabulary,
-)
-
-urlpatterns = [
-    url(r'^(?P<repository_id>[0-9]+)/vocabulary/$',
```

Comment:
Is this trailing slash intentional?  If so, does it represent a convention we want to follow in all URL patterns?